### ISSUE TITLE
Error during plugin initialization: gitbucket-ci-plugin-assembly-1.2.0.jar

### ISSUE THREAD

**Issue by k33g** [takezoe/gitbucket-ci-plugin#44]
Hey @takezoe 👋  I hope your trip was fine

I'm trying to make a clean install of GitBucket (4.18.0 and 4.17.0) + CI plugin and there is an error when loading plugins:

```
08:37:59.444 [Thread-14] INFO  g.core.plugin.PluginWatchThread - Shutdown PluginWatchThread
08:37:59.446 [qtp1705736037-9] INFO  gitbucket.core.plugin.PluginRegistry - Initialize gitbucket-notifications-plugin_2.12-1.2.0.jar
08:37:59.472 [qtp1705736037-9] INFO  gitbucket.core.plugin.PluginRegistry - Initialize gitbucket-ci-plugin-assembly-1.2.0.jar
08:37:59.529 [qtp1705736037-9] ERROR gitbucket.core.plugin.PluginRegistry - Error during plugin initialization: gitbucket-ci-plugin-assembly-1.2.0.jar
java.lang.NullPointerException: null
	at scala.collection.mutable.ArrayOps$ofRef$.length$extension(ArrayOps.scala:197)
	at scala.collection.mutable.ArrayOps$ofRef.length(ArrayOps.scala:197)
	at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:31)
	at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:29)
	at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:191)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
	at scala.collection.mutable.ArrayOps$ofRef.flatMap(ArrayOps.scala:191)
	at Plugin.$anonfun$versions$1(Plugin.scala:34)
	at io.github.gitbucket.solidbase.Solidbase.migrate(Solidbase.java:45)
	at io.github.gitbucket.solidbase.Solidbase.migrate(Solidbase.java:29)
	at gitbucket.core.plugin.PluginRegistry$.$anonfun$initialize$3(PluginRegistory.scala:284)
	at gitbucket.core.plugin.PluginRegistry$.$anonfun$initialize$3$adapted(PluginRegistory.scala:265)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at gitbucket.core.plugin.PluginRegistry$.initialize(PluginRegistory.scala:265)
	at gitbucket.core.plugin.PluginRegistry$.reload(PluginRegistory.scala:207)
	at gitbucket.core.controller.SystemSettingsControllerBase.$anonfun$$init$$15(SystemSettingsController.scala:222)
	at org.scalatra.ScalatraBase.liftAction(ScalatraBase.scala:285)
	at org.scalatra.ScalatraBase.$anonfun$invoke$1(ScalatraBase.scala:279)
	at org.scalatra.ApiFormats.withRouteMultiParams(ApiFormats.scala:189)
	at org.scalatra.ApiFormats.withRouteMultiParams$(ApiFormats.scala:175)
	at gitbucket.core.controller.ControllerBase.withRouteMultiParams(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.invoke(ScalatraBase.scala:278)
	at org.scalatra.ScalatraBase.invoke$(ScalatraBase.scala:277)
	at gitbucket.core.controller.ControllerBase.org$scalatra$json$JsonSupport$$super$invoke(ControllerBase.scala:34)
	at org.scalatra.json.JsonSupport.$anonfun$invoke$1(JsonSupport.scala:88)
	at org.scalatra.ApiFormats.withRouteMultiParams(ApiFormats.scala:189)
	at org.scalatra.ApiFormats.withRouteMultiParams$(ApiFormats.scala:175)
	at gitbucket.core.controller.ControllerBase.withRouteMultiParams(ControllerBase.scala:34)
	at org.scalatra.json.JsonSupport.invoke(JsonSupport.scala:82)
	at org.scalatra.json.JsonSupport.invoke$(JsonSupport.scala:81)
	at gitbucket.core.controller.ControllerBase.invoke(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.$anonfun$runRoutes$3(ScalatraBase.scala:253)
	at scala.Option.flatMap(Option.scala:171)
	at org.scalatra.ScalatraBase.$anonfun$runRoutes$1(ScalatraBase.scala:251)
	at scala.collection.immutable.Stream.flatMap(Stream.scala:490)
	at org.scalatra.ScalatraBase.runRoutes(ScalatraBase.scala:250)
	at org.scalatra.ScalatraBase.runRoutes$(ScalatraBase.scala:248)
	at gitbucket.core.controller.ControllerBase.runRoutes(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.runActions$1(ScalatraBase.scala:175)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$6(ScalatraBase.scala:187)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.cradleHalt(ScalatraBase.scala:205)
	at org.scalatra.ScalatraBase.executeRoutes(ScalatraBase.scala:187)
	at org.scalatra.ScalatraBase.executeRoutes$(ScalatraBase.scala:153)
	at gitbucket.core.controller.ControllerBase.executeRoutes(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.$anonfun$handle$1(ScalatraBase.scala:126)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withResponse(DynamicScope.scala:79)
	at org.scalatra.DynamicScope.withResponse$(DynamicScope.scala:77)
	at gitbucket.core.controller.ControllerBase.withResponse(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.$anonfun$withRequestResponse$1(DynamicScope.scala:59)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.withRequestResponse(DynamicScope.scala:58)
	at org.scalatra.DynamicScope.withRequestResponse$(DynamicScope.scala:56)
	at gitbucket.core.controller.ControllerBase.withRequestResponse(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.handle(ScalatraBase.scala:126)
	at org.scalatra.ScalatraBase.handle$(ScalatraBase.scala:122)
	at gitbucket.core.controller.ControllerBase.org$scalatra$servlet$ServletBase$$super$handle(ControllerBase.scala:34)
	at org.scalatra.servlet.ServletBase.handle(ServletBase.scala:53)
	at org.scalatra.servlet.ServletBase.handle$(ServletBase.scala:46)
	at gitbucket.core.controller.ControllerBase.org$scalatra$FlashMapSupport$$super$handle(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.$anonfun$handle$1(flashMap.scala:170)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.handle(flashMap.scala:145)
	at org.scalatra.FlashMapSupport.handle$(flashMap.scala:144)
	at gitbucket.core.controller.ControllerBase.handle(ControllerBase.scala:34)
	at org.scalatra.ScalatraFilter.$anonfun$doFilter$1(ScalatraFilter.scala:37)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.ScalatraFilter.doFilter(ScalatraFilter.scala:37)
	at org.scalatra.ScalatraFilter.doFilter$(ScalatraFilter.scala:32)
	at gitbucket.core.controller.ControllerBase.doFilter(ControllerBase.scala:73)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1759)
	at org.scalatra.ScalatraFilter.$anonfun$doNotFound$1(ScalatraFilter.scala:74)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$5(ScalatraBase.scala:178)
	at scala.Option.getOrElse(Option.scala:121)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$3(ScalatraBase.scala:178)
	at scala.Option.getOrElse(Option.scala:121)
	at org.scalatra.ScalatraBase.runActions$1(ScalatraBase.scala:178)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$6(ScalatraBase.scala:187)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.cradleHalt(ScalatraBase.scala:205)
	at org.scalatra.ScalatraBase.executeRoutes(ScalatraBase.scala:187)
	at org.scalatra.ScalatraBase.executeRoutes$(ScalatraBase.scala:153)
	at gitbucket.core.controller.ControllerBase.executeRoutes(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.$anonfun$handle$1(ScalatraBase.scala:126)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withResponse(DynamicScope.scala:79)
	at org.scalatra.DynamicScope.withResponse$(DynamicScope.scala:77)
	at gitbucket.core.controller.ControllerBase.withResponse(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.$anonfun$withRequestResponse$1(DynamicScope.scala:59)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.withRequestResponse(DynamicScope.scala:58)
	at org.scalatra.DynamicScope.withRequestResponse$(DynamicScope.scala:56)
	at gitbucket.core.controller.ControllerBase.withRequestResponse(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.handle(ScalatraBase.scala:126)
	at org.scalatra.ScalatraBase.handle$(ScalatraBase.scala:122)
	at gitbucket.core.controller.ControllerBase.org$scalatra$servlet$ServletBase$$super$handle(ControllerBase.scala:34)
	at org.scalatra.servlet.ServletBase.handle(ServletBase.scala:53)
	at org.scalatra.servlet.ServletBase.handle$(ServletBase.scala:46)
	at gitbucket.core.controller.ControllerBase.org$scalatra$FlashMapSupport$$super$handle(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.$anonfun$handle$1(flashMap.scala:170)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.handle(flashMap.scala:145)
	at org.scalatra.FlashMapSupport.handle$(flashMap.scala:144)
	at gitbucket.core.controller.ControllerBase.handle(ControllerBase.scala:34)
	at org.scalatra.ScalatraFilter.$anonfun$doFilter$1(ScalatraFilter.scala:37)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.ScalatraFilter.doFilter(ScalatraFilter.scala:37)
	at org.scalatra.ScalatraFilter.doFilter$(ScalatraFilter.scala:32)
	at gitbucket.core.controller.ControllerBase.doFilter(ControllerBase.scala:73)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1759)
	at org.scalatra.ScalatraFilter.$anonfun$doNotFound$1(ScalatraFilter.scala:74)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$5(ScalatraBase.scala:178)
	at scala.Option.getOrElse(Option.scala:121)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$3(ScalatraBase.scala:178)
	at scala.Option.getOrElse(Option.scala:121)
	at org.scalatra.ScalatraBase.runActions$1(ScalatraBase.scala:178)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$6(ScalatraBase.scala:187)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.cradleHalt(ScalatraBase.scala:205)
	at org.scalatra.ScalatraBase.executeRoutes(ScalatraBase.scala:187)
	at org.scalatra.ScalatraBase.executeRoutes$(ScalatraBase.scala:153)
	at gitbucket.core.controller.ControllerBase.executeRoutes(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.$anonfun$handle$1(ScalatraBase.scala:126)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withResponse(DynamicScope.scala:79)
	at org.scalatra.DynamicScope.withResponse$(DynamicScope.scala:77)
	at gitbucket.core.controller.ControllerBase.withResponse(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.$anonfun$withRequestResponse$1(DynamicScope.scala:59)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.withRequestResponse(DynamicScope.scala:58)
	at org.scalatra.DynamicScope.withRequestResponse$(DynamicScope.scala:56)
	at gitbucket.core.controller.ControllerBase.withRequestResponse(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.handle(ScalatraBase.scala:126)
	at org.scalatra.ScalatraBase.handle$(ScalatraBase.scala:122)
	at gitbucket.core.controller.ControllerBase.org$scalatra$servlet$ServletBase$$super$handle(ControllerBase.scala:34)
	at org.scalatra.servlet.ServletBase.handle(ServletBase.scala:53)
	at org.scalatra.servlet.ServletBase.handle$(ServletBase.scala:46)
	at gitbucket.core.controller.ControllerBase.org$scalatra$FlashMapSupport$$super$handle(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.$anonfun$handle$1(flashMap.scala:170)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.handle(flashMap.scala:145)
	at org.scalatra.FlashMapSupport.handle$(flashMap.scala:144)
	at gitbucket.core.controller.ControllerBase.handle(ControllerBase.scala:34)
	at org.scalatra.ScalatraFilter.$anonfun$doFilter$1(ScalatraFilter.scala:37)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.ScalatraFilter.doFilter(ScalatraFilter.scala:37)
	at org.scalatra.ScalatraFilter.doFilter$(ScalatraFilter.scala:32)
	at gitbucket.core.controller.ControllerBase.doFilter(ControllerBase.scala:73)
	at gitbucket.core.servlet.PluginControllerFilter$FilterChainWrapper.doFilter(PluginControllerFilter.scala:41)
	at gitbucket.core.servlet.PluginControllerFilter.doFilter(PluginControllerFilter.scala:33)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1759)
	at org.scalatra.ScalatraFilter.$anonfun$doNotFound$1(ScalatraFilter.scala:74)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$5(ScalatraBase.scala:178)
	at scala.Option.getOrElse(Option.scala:121)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$3(ScalatraBase.scala:178)
	at scala.Option.getOrElse(Option.scala:121)
	at org.scalatra.ScalatraBase.runActions$1(ScalatraBase.scala:178)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$6(ScalatraBase.scala:187)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.cradleHalt(ScalatraBase.scala:205)
	at org.scalatra.ScalatraBase.executeRoutes(ScalatraBase.scala:187)
	at org.scalatra.ScalatraBase.executeRoutes$(ScalatraBase.scala:153)
	at gitbucket.core.controller.ControllerBase.executeRoutes(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.$anonfun$handle$1(ScalatraBase.scala:126)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withResponse(DynamicScope.scala:79)
	at org.scalatra.DynamicScope.withResponse$(DynamicScope.scala:77)
	at gitbucket.core.controller.ControllerBase.withResponse(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.$anonfun$withRequestResponse$1(DynamicScope.scala:59)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.DynamicScope.withRequestResponse(DynamicScope.scala:58)
	at org.scalatra.DynamicScope.withRequestResponse$(DynamicScope.scala:56)
	at gitbucket.core.controller.ControllerBase.withRequestResponse(ControllerBase.scala:34)
	at org.scalatra.ScalatraBase.handle(ScalatraBase.scala:126)
	at org.scalatra.ScalatraBase.handle$(ScalatraBase.scala:122)
	at gitbucket.core.controller.ControllerBase.org$scalatra$servlet$ServletBase$$super$handle(ControllerBase.scala:34)
	at org.scalatra.servlet.ServletBase.handle(ServletBase.scala:53)
	at org.scalatra.servlet.ServletBase.handle$(ServletBase.scala:46)
	at gitbucket.core.controller.ControllerBase.org$scalatra$FlashMapSupport$$super$handle(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.$anonfun$handle$1(flashMap.scala:170)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:34)
	at org.scalatra.FlashMapSupport.handle(flashMap.scala:145)
	at org.scalatra.FlashMapSupport.handle$(flashMap.scala:144)
	at gitbucket.core.controller.ControllerBase.handle(ControllerBase.scala:34)
	at org.scalatra.ScalatraFilter.$anonfun$doFilter$1(ScalatraFilter.scala:37)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.ScalatraFilter.doFilter(ScalatraFilter.scala:37)
	at org.scalatra.ScalatraFilter.doFilter$(ScalatraFilter.scala:32)
	at gitbucket.core.controller.ControllerBase.doFilter(ControllerBase.scala:73)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1759)
	at gitbucket.core.servlet.TransactionFilter.$anonfun$doFilter$1(TransactionFilter.scala:39)
	at gitbucket.core.servlet.TransactionFilter.$anonfun$doFilter$1$adapted(TransactionFilter.scala:30)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.$anonfun$withTransaction$2(BlockingProfile.scala:207)
	at slick.JdbcProfileBlockingSession$BlockingSession.withTransaction(TransactionalJdbcBackend.scala:26)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.$anonfun$withTransaction$1(BlockingProfile.scala:207)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.withSession(BlockingProfile.scala:200)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.withTransaction(BlockingProfile.scala:207)
	at gitbucket.core.servlet.TransactionFilter.doFilter(TransactionFilter.scala:30)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1751)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:582)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:226)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1180)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:512)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1112)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:169)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:134)
	at org.eclipse.jetty.server.Server.handle(Server.java:534)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:320)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:283)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:108)
	at org.eclipse.jetty.io.SelectChannelEndPoint$2.run(SelectChannelEndPoint.java:93)
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.executeProduceConsume(ExecuteProduceConsume.java:303)
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produceConsume(ExecuteProduceConsume.java:148)
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.run(ExecuteProduceConsume.java:136)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:671)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:589)
	at java.lang.Thread.run(Thread.java:748)

```

I tried on linux and OSX - It's curious, my "my every day" installation is ok

**Comment by takezoe** [comment]
@k33g Our visit to Lyon was excellent! We are still in Paris and waiting for flight to Japan :-)

By the way, it's a migration bug in gitbcuket-ci-plugin 1.2.0 which causes with only clean installation. I fixed and released 1.2.1. Could you try it?
https://github.com/takezoe/gitbucket-ci-plugin/releases/tag/1.2.1

**Comment by k33g** [comment]
@takezoe have a safe trip

It works fine with 1.2.1 👍
